client/handle_http_test.go

```diff
@@ -206,7 +208,8 @@ func TestSlowTransfers(t *testing.T) {
 
 	// Make sure the errors are correct
 	assert.NotNil(t, err)
-	assert.IsType(t, &SlowTransferError{}, err)
+	require.IsType(t, &error_codes.PelicanError{}, err)
+	assert.Contains(t, err.Error(), "Transfer.SlowTransferError Error: Error code 602:")
```

Comment:
Is this representative of the error code an end user might see, or are we only checking part of the full error for the test? If the former, I worry it's not very descriptive and it has a confusing format.

Reply:
No, that is not the full error, just checking to see if this error contains the changes. The full error is this:
`Transfer.SlowTransfer Error: Error code 602: cancelled transfer, too slow.  Detected speed: 10.2 kB/s, total transferred: 153.6 kB, total transfer time: 15.000997891s`

Just can't really do much of the other check since we don't know the detected speed or the other numbers.